pkg/config/configenv/staging.go

```diff
@@ -67,6 +67,10 @@ var Staging = types.BacalhauConfig{
 				"/ip4/35.245.247.85/tcp/4001/p2p/12D3KooWEztGEJtqtzy7th2d7cTw2iR4CQCPHFUYvj66rhh9Cf7h",
 				"/ip4/35.245.247.85/udp/4001/quic/p2p/12D3KooWEztGEJtqtzy7th2d7cTw2iR4CQCPHFUYvj66rhh9Cf7h",
 			},
+			Profile:                "flatfs",
+			SwarmListenAddresses:   []string{"/ip4/0.0.0.0/tcp/0", "/ip6/::1/tcp/0"},
```

Comment:
These two addresses are not the same.  `::1` is the same address as 127.0.0.1, if we want to bind v6 to all v6 interfaces, we should use `::`